packages/replies/src/list/reply.tsx

```diff
@@ -190,6 +205,34 @@ export default function Reply({
     [setEditingMessage, asyncBack, back, push],
   )
 
+  const handleLikeReplyClick = async ({ messageId }: { messageId: string }) => {
+    await likeReply({ messageId })
+
+    setReactions((prev) => ({
+      ...prev,
+      like: {
+        count: (prev?.like?.count || 0) + 1,
+        haveMine: true,
+      },
+    }))
+  }
+
+  const handleUnLikeReplyClick = async ({
```

Comment:
`un-`접두사가 붙은 건 보통 한 단어로 치니까.. 

```suggestion
  const handleUnlikeReplyClick = async ({
```